net/munge/Portfile

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
```

Comment:
That's fine if we are indeed going with a sqash workflow. I want to bring it up before it's too late.